packages/aragon-api/src/index.js

```diff
@@ -332,6 +332,18 @@ export class AppProxy {
       pluck('result')
     )
   }
+
+  /**
+   * Allow apps to sign arbitrary data via a RPC call
+   *
+   * @param  {string} msgToSign
+   * @return {void}
+   */
+  signMessage (msgToSign) {
```

Comment:
I'd rename this method to `requestSignMessage()`, since the app ultimately doesn't control if the message gets signed or not.